src/components/App/content.tsx

```diff
@@ -62,9 +63,7 @@ function AppContentBase(): React.ReactElement {
         )}
       >
         {currentTab === 'Scheduler' && <Scheduler />}
-        {currentTab === 'Course details' && (
-          <span>Course Details Page Mobile placeholder</span>
-        )}
+        {currentTab === 'Course details' && <CourseDetails />}
```

Comment:
```suggestion
        {currentTab === 'Course Details' && <CourseDetails />}
```

Any particular reason why its not capitalized?

Reply:
It's this way in the Figma and while this should probably eventually be converted to an enum of some sort, for now we do string matching. And this is the tab name in the Figma.

Reply:
I agree capitalized looks better